Fix github and javadoc links (#44)
discord-jda/JDA-Website: docs/using-jda/making-a-music-bot.md

## Using LavaPlayer

> 1. [Set up LavaPlayer](https://github.com/Walkyst/lavaplayer-fork#readme)

Reviewer: ```suggestion
1. [Set up LavaPlayer](https://github.com/lavalink-devs/lavaplayer#readme)
```

2. Implement an AudioSendHandler 

>     - [Example](https://github.com/Walkyst/lavaplayer-fork#jda-integration)

Reviewer: ```suggestion
    - [Example](https://github.com/lavalink-devs/lavaplayer#jda-integration)
```

3. [Connect to a voice channel](#connecting-to-a-voicechannel)
4. [Register your AudioSendHandler](#sending-audio-to-an-open-audio-connection)

> 5. Use the LavaPlayer resources: [How To Use LavaPlayer](https://github.com/Walkyst/lavaplayer-fork#usage)

Reviewer: ```suggestion
5. Use the LavaPlayer resources: [How To Use LavaPlayer](https://github.com/lavalink-devs/lavaplayer#usage)
```
